devicon.json

```
@@ -5776,6 +5776,27 @@
         },
         "color": "#294172", 
         "aliases": [] 
-    }
+    },
+{
+        "name": "solidjs",
+        "tags": [
+            "nodejs",
+            "framework"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+	    	"plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+	    	"plain-wordmark",
+            ]
+        },
+        "color": "#85bce5",
```

Change to primary color from their website: https://www.solidjs.com/media
```suggestion
        "color": "#2c4f7c",
```